cmd/gctcli/commands.go

```diff
@@ -4,31 +4,33 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	"math"
 	"os"
 	"path/filepath"
+	"reflect"
+	"slices"
 	"strconv"
 	"strings"
 	"time"
 
 	"github.com/thrasher-corp/gocryptotrader/common"
 	"github.com/thrasher-corp/gocryptotrader/currency"
 	"github.com/thrasher-corp/gocryptotrader/exchanges/margin"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
 	"github.com/thrasher-corp/gocryptotrader/gctrpc"
 	"github.com/urfave/cli/v2"
 )
 
-var (
-	startTime, endTime, orderingDirection string
-	limit                                 int
-)
-
 var getInfoCommand = &cli.Command{
 	Name:   "getinfo",
 	Usage:  "gets GoCryptoTrader info",
 	Action: getInfo,
 }
 
+// error declarations
```

Comment:
🚧 Consider how this will look in godoc.
```suggestion
// Public Errors
```